aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/util/XmlAttributeSettingHelper.java

```diff
@@ -172,6 +172,9 @@ void setAttribute(Element element, T value) {
         }
         String stringifiedValue = value != null ? value.toString() : null;
         if (StringUtils.isBlank(stringifiedValue) || stringifiedValue.equals(ignoredValue)) {
+            if (element.hasAttribute(name)){
```

Comment:
Wow, love the one-liner solutions! Didn't event think that was so easy, thank you!
BTW, you don't need to check for `element.hasAttribute(name)` . Go ahead with `element.removeAttribute(...)` No exception if such attribute is not present

Reply:
Ok. Removed this checking